QuizApp/Quiz/QuizViewController.swift

```diff
@@ -1,11 +1,33 @@
 import Foundation
 import UIKit
+import SnapKit
+import Combine
 
 class QuizViewController: UIViewController {
 
+    var gradientBg: CAGradientLayer!
+    var titleLabel: UILabel!
+    var segmented: UISegmentedControl!
+
+    var quizListCollectionView: UICollectionView!
+
+    private var quizViewModel: QuizViewModel!
+    private var cancellables = Set<AnyCancellable>()
```

Comment:
Change order to follow the [style guide](https://github.com/fiveagency/ios-swift-guide/tree/master/Code%20Style#property-declaration-order).
`properties initialized in the same line as declared before those initialized at some other time`